cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,72 @@
+const { generateData } = require('./hermioneAccount.spec');
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with bank account', () => {
+    const {
+      depositAmount,
+      withdrawAmount,
+      balance,
+      user,
+      accountNumber,
+      balanceAfterWithdrow,
+      formattedDate
+    } = generateData();
+
+    cy.contains('.btn', 'Customer Login').click();
+    cy.get('[name="userSelect"]').select(user);
+    cy.contains('.btn', 'Login').click();
+
+    cy.contains('[ng-hide="noAccount"]', 'Account Number')
+      .contains('strong', accountNumber)
+      .should('be.visible');
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '5096')
+      .should('be.visible');
```

Comment:
The '5096' value seems to be hardcoded. Try to avoid hardcoding numbers or strings for tests if they are subject to change. Use dynamic references or variables instead.